README.md

```diff
@@ -1,8 +1,42 @@
+
+## Skills
+
+**Proficient:** Data Analysis, RDBMS/SQL, Linux/Bash Script, Git, Docker, Agile/Scrum, Cloud Computing [Google Cloud Platform (GCP)], R, HTML, MS Office, Excel
+
+**Competent:** Python, Javascript, Statistics, Finance, Accounting
+
+**Familiar:** Network Security, Software Security, Firewall Installation, Data Encryption, Web Development
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_JoelAttalla](https://github.com/jarviscanada/jarvis_data_eng_JoelAttalla)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_JoelAttalla/tree/master/linux_sql)]: Developed a monitoring system with integrated bash scripts and SQL queries to extract, store, and analyze data and it's processes across cluster nodes into a provisioned Postgres database.
+
+**Python Data Analytics [In Progress]** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_JoelAttalla/tree/master)]: Began the learning process for the development of a proof of concept (PoC) system to analyze customer shopping behavior, to provide data analytics to enhance a company's marketing strategies and decision making. 
+
+
+## Highlighted Projects
+**ITM 90A/B Graduation Project**: Developed and applied various IS solutions, diagrams, and models, using database, programming, security, and system designs to provide the client with their desired IS solutions to meet business needs.
+
+
+## Professional Experiences
+
+**Junior Software Developer, Jarvis Consulting (2021-Present)**: Learning and implementing various data engineering programs to attain the required experience for success in the workplace. Using Linux/SQL platforms, I have successfully developed a monitoring system with integrated bash scripts and SQL queries. Currently, I have begun the learning process for the development of a proof of concept (PoC) system using Python interphases.
```

Comment:
Learning and implementing various software and data projects to attain the required experience for success in the workplace. Agile, soft skills, responsibilities.